spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/SpannerOperations.java

```diff
@@ -169,12 +171,44 @@
 	 */
 	void update(Object object);
 
+	/**
+	 * Update an object in storage.
+	 * @param object the object to update.
+	 * @param includeColumns the columns to upsert. if none are given then all columns are
```

Comment:
nit: be more precise describing the includeColumns semantics.

Reply:
@balopat Thanks! added a commit to address this, but ironically that made the LGTM stale. PTAL again haha